static_src/stores/service_plan_store.js

```diff
@@ -62,6 +62,27 @@ class ServicePlanStore extends BaseStore {
         break;
       }
 
+      case serviceActionTypes.SERVICE_INSTANCES_RECEIVED: {
+        const instances = this.formatSplitResponse(action.serviceInstances);
+        this.fetched = false;
+        this.fetching = true;
+        this.emitChange();
+        const planRequests = [];
+        for (const instance of instances) {
+          planRequests.push(cfApi.fetchServicePlan(instance.service_plan_guid));
```

Comment:
Should we check to see if we have that service plan before making network requests? Or is there a risk that the data might have changed and therefore we always want to fetch the data?